docs/theming/developers.md

```diff
@@ -272,14 +280,15 @@ Context consumers are elements that passively consume the parent context.
 Elements such as `<rh-cta>`, `<rh-badge>` and `<rh-tag>` are examples of
 consumers.
 
-To make your element a color context consumer,
+To make your element a color context consumer:
 
-1. First `import` the classMap Lit directive, and the consumer controller
+1. First `import` the [classMap Lit directive](https://lit.dev/docs/templates/directives/#classmap),
+   and the consumer controller.
 2. Then add the `@colorContextConsumer()` decorator to a private property of
    `on` which is the type of `ColorTheme`.
-3. Add a classMap that implements the shadow class in your render method
-4. Use the theming tokens in your element's shadow styles, being sure.
-   to select the element which has the `.on.light`/`.on.dark` classes
+3. Add a classMap that implements the shadow class in your render method.
+4. Use the theming tokens in your element’s shadow styles, being sure
+   to select the element which has the `.on.light`/`.on.dark` classes.
```

Comment:
`which` → `that`

Reply:
[8ae3170](https://github.com/RedHat-UX/red-hat-design-system/pull/1937/commits/8ae31707ea11f29ae967709bb95afebae9820f44)